tests/test_files_sidebar_no_redraw_on_close_surface.py

```
@@ -0,0 +1,98 @@
+                try:
+                    client.close_workspace(workspace_id)
+                except Exception:
+                    pass
+
+
+if __name__ == "__main__":
+    try:
+        raise SystemExit(main())
+    except cmuxError as exc:
+        print(f"FAIL: {exc}")
+        raise SystemExit(1)
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)